### PR TITLE
We need this as well for proper 0.14 server support.

### PR DESCRIPTION
**Pull request by rstudner**
Have to expose ReactDOMServer as well for the server rendering per 0.14 upgrade docs.

### CODE REVIEW

**Comment by justin808** [comment]
We'll update this PR with the new gem version once we push that.

**Comment by justin808** [comment]
@rstudner @dylangrafmyre Let's update the react_on_rail gem soon and push to heroku.

**Comment by dylangrafmyre** [comment]
@justin808 what do you mean? The react_on_rails gem has already been updated to 1.0.0.pre as of last night for master.

**Comment by justin808** [comment]
Cool! Just trying to keep up! Closing this one! Rocking!

**Comment by rstudner** [comment]
Not sure why this was closed.  ReactDOM has been added to the client.base, but exposig ReactDOMServer still needs to go into the server.rails no?

(all this PR is for)

**Comment by justin808** [comment]
@rstudner You are correct. I was under the impression that @dylangrafmyre did this.
We also need the update to the server one as well: https://github.com/shakacode/react_on_rails/blob/master/spec/dummy/client/webpack.server.js#L30

Please rebase these changes on top of master, and if the build passes, let's merge.

I'm curious as to how the builds worked if this was not there.

**Comment by alexfedoseev** [comment]
> I'm curious as to how the builds worked if this was not there.

I think because old renderers are deprecated, not removed from core React package.

**Comment by rstudner** [comment]
@alexfedoseev is correct.  ReactDOM & ReactDOMServer are not *required* for 0.14, you just get deprecation warnings.  Thus, the build will pass just fine without the change.

**Comment by rstudner** [comment]
@justin808  rebased against master and pushed, build passed.

File: Gemfile.lock [hunk 1]
```diff
@@ -353,7 +353,7 @@ DEPENDENCIES
   rails-html-sanitizer
   rails_12factor
   rainbow
-  react_on_rails (~> 0.1.8)
+  react_on_rails (~> 1.0.0.pre)
```
Comment (justin808) [inline Gemfile.lock]: There's something wrong with this PR, as this is already on master: 

https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/Gemfile.lock#L356

@rstudner Any ideas?

**Comment by justin808** [comment]
@josiasds @dylangrafmyre This line is not right:
https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/webpack.client.base.config.js#L32

We should not be using ReactDom from the server.

See examples in this directory: https://github.com/shakacode/react_on_rails/tree/master/spec/dummy/client

https://github.com/shakacode/react_on_rails#additional-steps-for-server-rendering-option-prerender-shown-below

**Comment by justin808** [comment]
Sorry for the confusion @josiasds @dylangrafmyre, but my prior comment was incorrect. I just noticed that it is 'client.base' and not just 'base'. Do the line above is right.

**Pull request merged**